app/views/practices/contact.html.erb

```diff
@@ -20,11 +20,12 @@
                 <h2 class="text-normal margin-bottom-2">
                     This section helps people to reach out for support, ask questions, and connect about your practice.
                 </h2>
+
                 <%= nested_form_for(@practice, html: { multipart: true, style: 'max-width: 100%', class: 'usa-form', id: 'form'}) do |f| %>
                     <fieldset class="usa-fieldset">
                         <legend class="usa-sr-only">Practice Information</legend>
                         <div>
-                            <%= f.label :support_network_email, 'Email:', class: 'usa-label text-bold display-inline-block' %>&nbsp;<span>Type the email address dedicated to responding to messages for this practice.</span>
+                            <%= f.label :support_network_email, 'Email:', class: 'usa-label text-bold display-inline-block' %>&nbsp;<span>Type the email address dedicated to responding to messages for this practice. If you haven’t set up a practice’s distro email yet, enter the request under the "Your IT" portal linked on the home screen of your GFE.</span>
```

Comment:
nit: Based on the designs it looks like the `line-height` is `26px` 😅 
<img width="1155" alt="Screen Shot 2020-04-17 at 1 04 36 PM" src="https://user-images.githubusercontent.com/20211771/79600102-3f1fcb80-80ac-11ea-8448-c5ba36e4f346.png">